launch/sim_iarc7_safety.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
This launch file needs to disappear, and one needs to be made that just takes an argument with a list of nodes, like this from [here](http://wiki.ros.org/roslaunch/XML/rosparam#line-1-4):

Substitution allows you to make use of roslaunch args representing all or part of a YAML string. e.g.

    <arg name="whitelist" default="[3, 2]"/>
    <rosparam param="whitelist" subst_value="True">$(arg whitelist)</rosparam>